Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM ruby:2.4-stretch
+FROM ruby:2.6-alpine
+RUN apk update \
+    && apk upgrade \
+    && apk add --update --no-cache g++ musl-dev make libstdc++ yarn
```

Comment:
According to [eventmahcine's wiki](https://github.com/eventmachine/eventmachine/wiki/Building-EventMachine#alpine), this are the packages we need to install eventmachine in Alpine.